operations/security_and_authentication/Restrict_Access_to_NCN_Images_S3_Bucket.md

```diff
@@ -0,0 +1,724 @@
+         pkts      bytes target     prot opt in     out     source               destination   
+      ```
+
+22. Test connectivity after applying the ACL.
+
+      Re-run the connectivity test. While the results will be similar, they should all now be passing:
+
+      ```bash
+      ncn-m001# bash ./con_test.sh 
+      [i] MGMT RADOS over NMN
+      RADOS ncn-s001.nmn: PASS
+      RADOS ncn-s002.nmn: PASS
+      RADOS ncn-s003.nmn: PASS
+      [i] MGMT RADOS over HMN
+      RADOS ncn-s001.hmn: PASS
+      RADOS ncn-s002.hmn: PASS
+      RADOS ncn-s003.hmn: PASS
+      [i] MGMT RADOS over CMN
+      RADOS ncn-s001.cmn: PASS
+      RADOS ncn-s002.cmn: PASS
+      RADOS ncn-s003.cmn: PASS
+      [i] MGMT HAProxy over NMN
+      HAPROXY (CEPH) HTTP ncn-s001.nmn: PASS
+      HAPROXY (CEPH) HTTPS ncn-s001.nmn: PASS
+      HAPROXY (CEPH) HTTP ncn-s002.nmn: PASS
+      HAPROXY (CEPH) HTTPS ncn-s002.nmn: PASS
+      HAPROXY (CEPH) HTTP ncn-s003.nmn: PASS
+      HAPROXY (CEPH) HTTPS ncn-s003.nmn: PASS
+      ```
+
+23. Validate no connection can be made to HAProxy for `ncn-images` or Ceph RADOS GW (at all) from compute nodes and UANs.
+
+      Use `rgw-vip` as it will resolve to one of the storage nodes.
+
+      ```bash
+      nid000002# host rgw-vip
+      rgw-vip has address 10.252.1.3
+
+      nid000002# curl http://rgw-vip/ncn-images/
+      <html><body><h1>403 Forbidden</h1>
+      Request forbidden by administrative rules.
+      </body></html>
+
+      nid000002# curl -k https://rgw-vip/ncn-images/
+      <html><body><h1>403 Forbidden</h1>
+      Request forbidden by administrative rules.
+      </body></html>
+
+      nid000002# curl --connect-timeout 2 rgw-vip:8080
+      curl: (28) Connection timed out after 2001 milliseconds
+      ```
+
+      Look for a 403 response in the HAProxy logs:
+
+      ```bash
+      ncn-m001# pdsh -N -w ncn-s00[1-4] "cd /var/log && zgrep -h -i -E 'haproxy.*frontend' messages || exit 0" | grep "ncn-images"
+      2022-07-13T13:57:08+00:00 xxx-ncn-s001.local haproxy[43591]: 10.252.1.13:50238 [13/Jul/2022:13:57:08.363] http-rgw-frontend http-rgw-frontend/<NOSRV> 0/-1/-1/-1/0 403 212 - - PR-- 1/1/0/0/0 0/0 "GET /ncn-images/ HTTP/1.1"
+      2022-07-13T14:01:11+00:00 xxx-ncn-s001.local haproxy[43591]: 10.252.1.13:50240 [13/Jul/2022:14:01:11.038] http-rgw-frontend http-rgw-frontend/<NOSRV> 0/-1/-1/-1/0 403 212 - - PR-- 1/1/0/0/0 0/0 "GET /ncn-images/ HTTP/1.1"
+      ...
+      ```
+
+      In the firewall logs, the Ceph RADOS GW traffic will be dropped on the storage node. For example:
+
+      ```bash
+      ncn-m001# pdsh -N -w ncn-s00[1-4] "grep RADOSGW /var/log/firewall"
+      2022-07-13T14:02:03.418750+00:00 xxx-ncn-s001 kernel: [4397628.546654] RADOSGW-DROPIN=bond0.nmn0 OUT= MAC=b8:59:9f:f9:1d:22:a4:bf:01:3f:6f:91:08:00 SRC=10.252.1.13 DST=10.252.1.3 LEN=52 TOS=0x00 PREC=0x00 TTL=64 ID=9727 DF PROTO=TCP SPT=59278 DPT=8080 WINDOW=42340 RES=0x00 SYN URGP=0 
```

Comment:
The bond0.nmn0 is also not correct for a 1.0 system.   That should be vlan002, I believe.